tests/test_api.py

```diff
@@ -0,0 +1,53 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: 
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import os
+from unittest.mock import Mock,patch,mock_open
+
+from nose.tools import assert_is_not_none
+
+from src import api
+
+
+def test_get_request():
+    """
+    To test api when an image is being downloaded
+    """
+    with patch('src.api.requests.get') as mock_get:
+        mock_get.return_value.ok = True
+        url = 'https://raw.githubusercontent.com/lk-geimfari/unittest/master/logo.png'
+        response = api.get_request(url)
+    assert_is_not_none(response)
```

Comment:
Use `assert response is not None` instead and do not use `nose`. That said, I am not sure what you are testing in this cases: can you explain a bit?

Reply:
I am just testing that the URL I am hitting does not give me an empty response, if it does give then I can not fetch the data from that URL